pkg/blob/blob.go

```diff
@@ -317,7 +317,7 @@ func checkContainerNameBeginAndEnd(containerName string) bool {
 
 // isSASToken checks if the key contains the patterns. Because a SAS Token must have these strings, use them to judge.
 func isSASToken(key string) bool {
-	return strings.Contains(key, "?sv=")
+	return strings.HasPrefix(key, "?")
```

Comment:
is the previous test failure due to this issue?

Reply:
It's one of the reason. The other one is my mistake.